apps/webapp/app/tailwind.css

```
@@ -223,6 +224,84 @@
+   status combo components. */
+:is([data-theme="dark"], [data-theme="light"]) .system-mono-label {
+  color: inherit;
+}
+
+/* Tinted status chips respond to the contrast control: a ring in the chip's
+   own text color fades in as contrast rises, so the soft tint keeps its
+   footprint but the chip gains definition. Transparent at contrast 0;
+   Classic never sets the variable. Marker set by the chip components. */
+:is([data-theme="dark"], [data-theme="light"]) .contrast-chip {
+  box-shadow: inset 0 0 0 1px
+    color-mix(in srgb, currentColor calc(var(--theme-contrast, 0) * 70%), transparent);
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)